mono/metadata/threads-types.h

```diff
@@ -464,137 +464,10 @@ mono_thread_internal_describe (MonoInternalThread *internal, GString *str);
 gboolean
 mono_thread_internal_is_current (MonoInternalThread *internal);
 
-gboolean
-mono_threads_is_current_thread_in_protected_block (void);
```

Comment:
Was the removal of all these APIs intentional?

Reply:
No, it looks like I messed up the merge when I cherry-pick the changes. I'll fix this.